### DISCUSSION TITLE
Is the format of the .blk file some kind of specification or is it customised by Gaijin?

### DISCUSSION THREAD

**Discussion by axiangcoding** [Warthunder-Open-Source-Foundation/wt_ext_cli#18] [General]
I want to parse and modify the configuration file (e.g. enable some configuration items), I think it's better to parse the blk file and append the content to it.

**Comment by FlareFlo** [comment]
This repo has (russian) docs for the file format https://github.com/kotiq/blk

**Comment by FlareFlo** [comment]
Confirmed source: https://gaijinentertainment.github.io/DagorEngine/dagor-tools/blk/index.htmo
